tabiya-tech/taxonomy-model-application
# feat: app UI/UX implementation

### What does this PR do?

- move taxonomy model app setup under /frontend/app direcotry
- reconfigure the routing to support layout context
- app directory folder stracture
- introduce app layout out provider to faciliate dynamic rendering of its components

### How can it be tested
You should be able to run the server `yarn start` and welcome page rendered, and be able to navigate to the following route:
- `/#/info`
- `/#/modeldirectory`
